doc/cephfs/cephfs-top.rst

```diff
@@ -63,6 +63,60 @@ By default, `cephfs-top` uses `client.fstop` user to connect to a Ceph cluster::
   $ ceph auth get-or-create client.fstop mon 'allow r' mds 'allow r' osd 'allow r' mgr 'allow r'
   $ cephfs-top
 
+Description of Fields
+---------------------
+
+chit : cap hit
+Percentage of caps issued over total number of caps
+
+dlease : dentry lease
+Percentage of dentry leases handed out over the total dentry lease requests
+
+ofiles : opened files
+Number of opened files
+
+oicaps : pinned caps
+Number of pinned caps
+
+oinodes : opened inodes
+Number of opened inodes
+
+rtio : total size of read IOs
+Number of bytes read in input/output operations generated by all process
+
+wtio : total size of write IOs
+Number of bytes written in input/output operations generated by all processes
+
+raio : average size of read IOs
+Mean of number of bytes read in input/output operations generated by all process over total IO done
+
+waio: average size of write IOs
+Mean of number of bytes written in input/output operations generated by all process over total IO done
+
+rsp : read speed
+Speed of read IOs compared with the last refresh
```

Comment:
What does "last refresh" mean?

Reply:
I took it from https://github.com/ceph/ceph/commit/ec22208364086605361c2f7f54f6b9908fa8bc7e#diff-c739b905b3b881e50bd7523f0f5c602709f7b7bf30881d0e17b07e5c9524741cR84 . I suppose it means with respect to the duration from the last `refresh_client`.

Reply:
yeh, that's a code comment which is meant to refer code blocks. We should be _much_ clear in the docs.

Reply:
How about now, PTAL?